### DISCUSSION TITLE
Make it possible to "rerun" a backport

### DISCUSSION THREAD

**Discussion by jackkoenig** [Mergifyio/mergify#5099] [Ideas]
When you have PRs that depend on each other, you can run into conflict issues where the dependent PR is backported before the dependee backport is merged.

For example, say I have:
PR#1 (main)
PR#2 (main) - depends on PR#1
PR#1 merged
PR#3 (stable) (backport of PR#1) created
PR#2 merged
PR#4 (stable) (backport of PR#2) created -- conflict (because PR#3 has not yet been merged to stable)

In this situation, it would be useful to manually "rerun" the PR#4 backport, ideally with a command. Better yet would be some way of tracking dependencies between PRs and thus between their backports automatically, but I understand that is a much more complicated ask.

Is there already a way to do this? Or is this a reasonable feature request?

**Comment by jd** [comment]
You can create a backport using a command: https://docs.mergify.com/commands/#backport
It's therefore possible to re-create backports once PR#3 is merged if needed.

Mergify can track dependencies between PRs, but unfortunately, the backport mechanism has no knowledge of that. I think the problem would then be to know if the dependencies have to be or not backported, which is not an easy task.
